internal/services/pam/testdata/TestSetDefaultBrokerForUser/set-default-broker.db

```diff
@@ -2,29 +2,35 @@ GroupByID:
   "11111": '{"Name":"group1","GID":11111}'
   "22222": '{"Name":"group2","GID":22222}'
   "33333": '{"Name":"group3","GID":33333}'
+  "44444": '{"Name":"group4","GID":44444}'
   "99999": '{"Name":"commongroup","GID":99999}'
 GroupByName:
   commongroup: '{"Name":"commongroup","GID":99999}'
   group1: '{"Name":"group1","GID":11111}'
   group2: '{"Name":"group2","GID":22222}'
   group3: '{"Name":"group3","GID":33333}'
+  group4: '{"Name":"group4","GID":44444}'
 GroupToUsers:
   "11111": '{"GID":11111,"UIDs":[1111]}'
   "22222": '{"GID":22222,"UIDs":[2222]}'
-  "33333": '{"GID":33333,"UIDs":[4444]}'
-  "99999": '{"GID":99999,"UIDs":[2222,4444]}'
+  "33333": '{"GID":33333,"UIDs":[3333]}'
+  "44444": '{"GID":44444,"UIDs":[4444]}'
+  "99999": '{"GID":99999,"UIDs":[2222,3333,4444]}'
 UserByID:
   "1111": '{"Name":"userwithbroker","UID":1111,"GID":11111,"Gecos":"userwithbroker gecos\nOn multiple lines","Dir":"/home/userwithbroker","Shell":"/bin/bash","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
   "2222": '{"Name":"userwithinactivebroker","UID":2222,"GID":22222,"Gecos":"userwithinactivebroker","Dir":"/home/userwithinactivebroker","Shell":"/bin/dash","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
-  "4444": '{"Name":"usersetbroker","UID":4444,"GID":33333,"Gecos":"usersetbroker","Dir":"/home/usersetbroker","Shell":"/bin/zsh","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
+  "3333": '{"Name":"usersetbroker","UID":3333,"GID":33333,"Gecos":"usersetbroker","Dir":"/home/usersetbroker","Shell":"/bin/dash","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
```

Comment:
why do you change usersetbroker to 3333? You can maybe keep it to 4444 and insert the local user to 3333 (I did not see it being reused in any other part of the program, so it’s not for consistency there either, no?)

Reply:
The mismatching UID/GID was bothering me too much :joy:

Reply:
that’s an acceptable argument :)